Author: Andreas Beckmann <anbe@debian.org>
Description: dump the generated conftest headers

--- a/Kbuild
+++ b/Kbuild
@@ -126,6 +126,16 @@ NV_CONFTEST_HEADERS += $(obj)/conftest/h
 NV_CONFTEST_HEADERS += $(NV_CONFTEST_COMPILE_TEST_HEADERS)
 
 
+.PHONY: conftest-verbose
+conftest-verbose:
+	@echo 'NV_CONFTEST_CMD=$(NV_CONFTEST_CMD)'
+	@echo 'NV_CONFTEST_CFLAGS=$(NV_CONFTEST_CFLAGS)'
+	@echo 'KBUILD_CFLAGS=$(KBUILD_CFLAGS)'
+	@echo 'LINUXINCLUDE=$(LINUXINCLUDE)'
+	@echo 'LDFLAGS=$(LDFLAGS)'
+	@echo 'KBUILD_LDFLAGS=$(KBUILD_LDFLAGS)'
+	@echo 'ARCH=$(ARCH)'
+
 #
 # Generate a header file for a single conftest compile test. Each compile test
 # header depends on conftest.sh, as well as the generated conftest/headers.h
@@ -150,6 +160,8 @@ define NV_GENERATE_COMPILE_TEST_HEADER
 	@mkdir -p $(obj)/conftest
 	@# concatenate /dev/null to prevent cat from hanging when $$^ is empty
 	@cat $$^ /dev/null > $$@
+	@echo " CONFTEST $$(notdir $$@):"
+	@cat $$@
 endef
 
 #
 
